github/repos_test.go

```diff
@@ -109,7 +109,8 @@ func TestRepositoriesService_ListByOrg(t *testing.T) {
 
 	mux.HandleFunc("/orgs/o/repos", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		testHeader(t, r, "Accept", mediaTypeLicensesPreview)
+		testInHeader(t, r, "Accept", mediaTypeLicensesPreview)
+		testInHeader(t, r, "Accept", mediaTypeCodesOfConductPreview)
```

Comment:
This tests for presence of wanted media types in accept header, but not for absence of unwanted ones.

I'd suggest just using the existing `testHeader` helper. Similar to how it's done here:

https://github.com/google/go-github/blob/d20deacea658223895efa3600c28053e733a512f/github/repos_test.go#L233